flowclient/flowclient/client.py

```diff
@@ -935,3 +935,34 @@ def flows(
         "to_location": to_location,
         "aggregation_unit": aggregation_unit,
     }
+
+
+def location_introversion(
+    start_date: str, stop_date: str, aggregation_unit: str, direction: str = "both"
```

Comment:
In flowclient, the exposed parameter should be called `end_date`:
```suggestion
    start_date: str, end_date: str, aggregation_unit: str, direction: str = "both"
```

Reply:
Fixed (and I'll fix that in TotalNetworkObjects too)